packages/webapp/package.json

```diff
@@ -31,6 +34,8 @@
         "esbuild-plugin-svgr": "1.0.1",
         "esbuild-sass-plugin": "2.2.4",
         "html-react-parser": "1.4.8",
+        "jest-css-modules-transform": "4.4.2",
+        "jest-sonar-reporter": "2.0.0",
```

Comment:
Hi @unseen1980, we have `jest-sonar` module as `devDependency` in root `package.json`. In PR https://github.com/SAP/guided-answers-extension/pull/113 I'm currently working on enabling SonarCloud scans and modify the `jest.config.js` files to use `jest-sonar`.
Do we also need `jest-sonar-reporter` ?

Reply:
No we don't, I added it because of 
```testResultsProcessor: 'jest-sonar-reporter',```. I'll remove it.